src/components/field-key/row.vue

```diff
@@ -69,12 +71,6 @@ export default {
   methods: {
     showCode() {
       this.$emit('show-code', this.fieldKey, this.$refs.popoverLink);
-    },
-    revoke() {
-      // Bootstrap does not truly disable dropdown menu items marked as
-      // disabled.
-      if (this.fieldKey.token == null) return;
-      this.$emit('revoke', this.fieldKey);
```

Comment:
We no longer need the `if` because `useDisabled()` will stop the handler from being called. Without the `if`, the method is just one line, so I moved it into the template.